README.md

```diff
@@ -17,6 +17,24 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
 **Note:** The date shown is the date that the change was released to Sandbox or Production.
 
 ---
+### 25 April 2025 ###
+
+#### Self Assessment Accounts API ####
+
+#### Removed ####
+
+The `INCONSISTENT_QUERY_PARAMS` and `INVALID_DATE_RANGE` Gov-Test-Scenarios are no longer available in the Sandbox for the following endpoint:
+
+- Retrieve Self Assessment Balance and Transactions
+
+The errors that these test scenarios represented can now be triggered in the sandbox.
+
+The `INCONSISTENT_QUERY_PARAMS` error can be triggered in the following scenarios:
+
+- providing values for `fromDate` and `toDate`, or `docNumber` (or all three), whilst `onlyOpenItems` is true
+- providing no values the `fromDate` and `toDate`, or `docNumber` (or none of them), whilst `onlyOpenItems` is false (omitting the `onlyOpenItems` parameter will default its value to false)
```

Comment:
should this be `providing no values for the` rather than `providing no values the`

Reply:
I think it should actually be `providing no values for` rather than `providing no values for the`